frontend/src/App.tsx

```diff
@@ -23,14 +24,14 @@ function App() {
         fetchAllFlights();
     },[])
 
-
     return (
         <div className={"app"}>
             <Header />
             <main>
                 <Routes>
                     <Route path={"/"} element={<Home />}/>
                     <Route path={"/flight"} element={<FlightPage data={flightData}/>}/>
+                    <Route path={"/flight/add"} element={<AddFlightPage fetchAllFlights={fetchAllFlights}/>}/>
```

Comment:
I think u can achieve what u are looking for if u get the useEffect with the useState and the fetchAllFlights and transfer all this to the /flight page...